app/gui2/src/components/CircularMenu.vue

```diff
@@ -90,8 +90,8 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
       />
       <ToggleIcon
         icon="record"
-        class="overrideRecordingButton slot7"
-        data-testid="overrideRecordingButton"
+        class="slot7 record"
```

Comment:
What is the reason for this change? It’s not mentioned in PR description, and `record` class is not used in CSS rules.

Reply:
The record class is defined inside `ToggleIcon` intentionally. I just moved it around a bit, because the style was duplicated.  An alternative to this would be to make a prop for "button type", and select styles based on that. If we had a few more of them, I'd prefer that. But current approach is simpler and we don't need anything more complicated for icons now.